lib/phoenix/code_reloader/colors.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Phoenix.CodeReloader.Colors do
+  defp css_color(n) when is_binary(n),
+    do: String.to_integer(n) |> css_color()
+  defp css_color(31), do: :red
+  defp css_color(33), do: "#b3b30a"
+
+  def to_html(string) do
+    string = Plug.HTML.html_escape(string)
```

Comment:
We are already calling `html_escape` on the input before calling this function. Won't this lead to double escaping?